email-log.php

```diff
@@ -76,26 +129,71 @@ function __construct() {
 		load_plugin_textdomain( 'email-log', false, $this->translations );
 
 		// Register hooks
+
+		/**
+		 * Fires after the basic admin menu structure is in place.
+		 *
+		 * @since Genesis
+		 */
 		add_action( 'admin_menu', array( $this, 'register_settings_page' ) );
```

Comment:
Calls to `add_action` and `add_filter` need not have a PHPDoc block.

PHPDoc block is needed only for `apply_filter` and `do_action` functions.